### PR TITLE
[master] Updated Clustered Singleton.adoc

### PR DESCRIPTION
**Pull request by felixif**
I updated the information on the page, recommending users set the `serialVersionUID` to avoid possible issues at re-deployment.

### CODE REVIEW

File: docs/modules/ROOT/pages/Technical Documentation/Public API/Clustered Singleton.adoc [hunk 1]
```diff
@@ -41,6 +41,15 @@ It is highly recommended to enable distributed locks for CDI Clustered beans: `@
 . Data corruption is possible if distributed lock is turned off and there is a race condition between method calls to the same bean on multiple cluster nodes simultaneously. In such a case, last method call wins the singleton state
 . Clustered singletons are stored in Hazelcast map under "Payara/(EJB/CDI)/singleton/<singletonComponentName>/[lock]"
 
+It is also highly recommended to add a `serialVersionUID` variable to the `@Clustered @Singleton` class, in order to remember versions of the `@Clustered @Singleton` class to verify that a loaded class and the serialized object are compatible.
```
Comment (AlanRoth) [inline docs/modules/ROOT/pages/Technical Documentation/Public API/Clustered Singleton.adoc:44]: Could be put in a IMPORTANT or TIP admonition, what do you think?

Comment (felixif) [reply docs/modules/ROOT/pages/Technical Documentation/Public API/Clustered Singleton.adoc:44]: Yep, I agree! I will update it now.

File: docs/modules/ROOT/pages/Technical Documentation/Public API/Clustered Singleton.adoc [hunk 2]
```diff
@@ -41,6 +41,16 @@ It is highly recommended to enable distributed locks for CDI Clustered beans: `@
 . Data corruption is possible if distributed lock is turned off and there is a race condition between method calls to the same bean on multiple cluster nodes simultaneously. In such a case, last method call wins the singleton state
 . Clustered singletons are stored in Hazelcast map under "Payara/(EJB/CDI)/singleton/<singletonComponentName>/[lock]"
 
+[IMPORTANT]
+It is also highly recommended to add a `serialVersionUID` variable to the `@Clustered @Singleton` class, in order to remember versions of the `@Clustered @Singleton` class to verify that a loaded class and the serialized object are compatible.
```
Comment (fturizo) [inline docs/modules/ROOT/pages/Technical Documentation/Public API/Clustered Singleton.adoc:45]: A bit redundant to say `@Clustered @Singleton` twice: Once is enough. Also, this paragraph should state:

"that a previous serialized entry in the Data Grid is compatible with updated versions of the same class".

**Review by fturizo — CHANGES_REQUESTED** [review]
Couple of changes needed.

**Comment by fturizo** [comment]
@felixif, please don't forget to make an equivalent port of this PR to `payara6-master`.

**Review by fturizo — APPROVED** [review]

**Pull request merged**